airflow/www/extensions/init_views.py

```diff
@@ -73,8 +73,12 @@ def init_plugins(app):
     appbuilder = app.appbuilder
 
     for view in plugins_manager.flask_appbuilder_views:
-        log.debug("Adding view %s", view["name"])
-        appbuilder.add_view(view["view"], view["name"], category=view["category"])
+        try:
+            log.debug("Adding view %s with menu", view["name"])
+            appbuilder.add_view(view["view"], view["name"], category=view["category"])
+        except KeyError:
```

Comment:
Multiple keys can cause this error. This does not necessarily mean that the `name` key is missing. What do you think to explicitly check the presence of the `name` key?

Reply:
The outcome should be the same though: if `name`, `category` or `view` is missing a `KeyError` is raised and the other path is ran -- if a missing `view` key _was the cause_ of the exception then it will be raised (again) below.

Maybe not the clearest from an error point of view but I think it still satisfies the intent?

Reply:
This is not very important, but I would prefer the intent of the code to be clearly stated. This makes it easier to maintain the code in the long run because you don't have to think about what a particular piece of code is doing.

I could ignore it if the change had tests and documentation.Now I am asking you to make these changes, add missing tests and documentation.  I would very much like this change to be completed, but as an OSS project, we also need to keep this code easy to maintain in the long term.

Reply:
@mik-laj got it! 👍🏻
I didn't use if-else paradigm to check the key since it is against the python's EAFP principle. I can still check against the missing key which threw error though, in the catch block so will add that asap.

I've been setting up test environment too and will add the required tests asap.